### PR TITLE
fixes #336 - send process param to random probes

### LINKED ISSUE #336: the random probe API should take the same values as `/api/v1/data`
At the moment, the probes we are showing have hard-coded defaults. As such, if you click on one of htem, you then send a request to `/api/v1/data` that might not necessarily be there, since the params we pass into `/api/v1/data` might not be applicable.

The solution should be straightforward – we need to make sure the random probe API takes in the exact same params as `/api/v1/data`

### PR DESCRIPTION
**Pull request by spasovski**
This should be the right approach and I no longer get "missing data" probes on the Home view.

### CODE REVIEW

File: glam/tests/test_api.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def _create_aggregation(self, name="gc_ms", data=None, multiplier=1.0):
             "version": "72",
             "os": "Windows",
             "build_id": "*",
-            "process": 0,
+            "process": 2,
```
Comment (robhudson) [inline glam/tests/test_api.py]: Can you use the constant here? `constants.PROCESS_CONTENT`

**Review by robhudson — APPROVED** [review]
r+wc

File: src/routing/pages/Home.svelte [hunk 2]
```diff
@@ -9,6 +9,8 @@
   import { store, currentQuery } from '../../state/store';
   import { getRandomProbes } from '../../state/api';
   import { getPath } from '../Router.svelte';
+
+  const NUMBER_OF_RANDOM_PROBES = 9;  // TODO: add this to the upcoming config.js
```
Comment (openjck) [inline src/routing/pages/Home.svelte]: New eslint error:

```
  13:37  error  Multiple spaces found before '// TODO: add t...'  no-multi-spaces
```

Comment (openjck) [reply src/routing/pages/Home.svelte]: 1337

**Comment by openjck** [comment]
Otherwise looks good to me!

**Pull request merged**